packages/gatsby-plugin-now/gatsby-node.js

```diff
@@ -26,8 +22,33 @@ exports.onPostBuild = async ({ store }) => {
     }
   }
 
+  const finalRoutes = [
+    {
+      src: '/(static|icons)/(.*)',
+      headers: { 'cache-control': 'public,max-age=31536000,immutable' },
+      continue: true,
+    },
+    {
+      src: '/.*\\.(js|json|css)$',
+      headers: { 'cache-control': 'public,max-age=31536000,immutable' },
+      continue: true,
+    },
+    {
+      src: '/.*\\.html',
+      headers: { 'cache-control': 'public,max-age=0,must-revalidate' },
+      continue: true,
+    },
+    {
+      src: '/page-data/.*',
+      headers: { 'cache-control': 'public,max-age=0,must-revalidate' },
+      continue: true,
+    },
+    ...routes,
+    { src: '/.*', status: 404, dest: '/404' },
```

Comment:
```suggestion
    { src: '/.*', status: 404, dest: '/404.html' },
```

Should this be an html page?